src/twisted/internet/base.py

```diff
@@ -736,14 +737,14 @@ def removeSystemEventTrigger(self, triggerID):
         self._eventTriggers[eventType].removeTrigger(handle)
 
 
-    def callWhenRunning(self, _callable, *args, **kw):
+    def callWhenRunning(self, f: Callable[..., Any], *args, **kw):
```

Comment:
this is a signature change same as addSystemEventTrigger ... this should be named `callable`

Reply:
I took this signature change out